packages/core/src/components/popover-canvas/readme.md

```diff
@@ -27,20 +27,44 @@ use the `referenceEl` prop rather than the `selector` the referenced element can
 </TdsPopoverCanvas>
 
 ```
+
+## Controlled vs Uncontrolled
+
+- **Controlled**: 
+  How does component enter controlled mode?
+  - Leaving `selector` or `referenceEl` prop empty
```

Comment:
I think the users should still be able to add a `referenceEl` to get the correct positioning.

This is how I imagine it:

```jsx
<button id="my-id" onClick={() => showPopover = !showPopover}>
 Open/close popover
</button>


<tds-popover-canvas selector="#my-id" show={showPopover}>
 // Content
</tds-popover-canvas>
```

So I think controlled should **only** case about if `show` is not null. If it is set to `true` or `false` the internal logic should be completely ignore appart from setting the correct reference/selector. 

Maybe this isn't possible with the current setup and if so I would say ignore my silly comments for now. But I do this the long term goal for all "show/hide-components" to have either a controlled or uncontrolled state.

Reply:
I get you, I think I need to rephrase it. Think is if users do not pass the selector or reference, a component can not initialize properly, therefore we have to block logic from running, otherwise, it will spit out errors in the console, first for missing reference and then for missing instance to run update script from popper. 

So, users can use selector/referenceEl and open prop today too, I think what I need to clarify is that for the right positioning of component, selector/reference are mandatory.

Reply:
Ah okey! Yeah that sounds good. If no referenceEl or selector is passed we shouldn't really render anything, right? Instead just log an error.

Reply:
In that case, user will get a warning that component's internal logic is disabled. Component wont show up until user toggles open to true.